proposals/NNNN-Persisting HMI Capabilities specific to headunit.md

```diff
@@ -0,0 +1,89 @@
+
+
+## Potential downsides
+
+This change is prone to errors by HMI. If HMI sends incorrect HMI capabilities during first boot up/after master reset, then that cannot be corrected until a master reset is performed. But we do expect HMI to send correct HMI capabilities back at all times. Therefore, this should not be an issue as long as HMI ensures that only correct values are sent to SDL core. We should still put this as best practices/warning in HMI guidelines.
+
+## Impact on existing code
+
+There will be changes in bootup initialization sequence around all HMI capabilities. Core waits on certain notifications/responses to send requests for HMI capabilities, all that logic would need to be reworked. This rework should yield a better bootup time for core and better app resumption behavior.
+
+## Alternatives considered
+
+HMI can potentially take long time to respond with HMI capabilities, that would mean if an app registers before HMI has sent response back to core then app would get incorrect capabilities. To avoid this, core can choose to wait before sending RAI response to the app. However, that has potential to break app resumption use case and would possibly cause incorrect app behavior.
+
+## Additional Comments
+
+1. There are recent changes to `SystemCapabilities` in _Widgets_Proposal_ and _App_Services_ proposal. For those, since apps get `onSystemCapability` notification upon registration with capabilities specific to template selected/services provided by app(s), this information is NOT static and would change over the course of app registration itself. That means that when app changes template(layout) or another apps register as service providers then the capabilities struct will change along with that. Hence there is NO super set of DisplayCapability/AppService which would be delivered to the app or core during SDL bootup/app registration.
+
+There are two points to consider still:
+
+   * HMI may still be slow in sending `onSystemCapabiity` -> `DisplayCapability` notification to the app. To circumvent that case, we can recommend adding this information as part of hashId persistence for the app. Since the template/capabilities are specific to the app. It makes sense to persist it on per app basis. This should be a separate proposal.
+   * `UI.GetCapabilities` for `DisplayCapabilities` is being deprecated due to WidgetProposal. But the implementation will still be there for next core version. So we can keep this in the proposal as there will be a transition time period for the apps to new setup.
+
+2. There will be multiple writes to _cache_ file during first boot up, also there will be atlease three writes upon language change. To make write process more efficient, all this data can be cached first along with a timer which checks if internal cache has changed before writing to file. Similar process is followed during PT persitence. This is however is implementation detail and can be decided during implementation itself.
```

Comment:
```suggestion
2. There will be multiple writes to _cache_ file during first boot up, also there will be at least three writes upon language change. To make the write process more efficient, all this data can be cached first along with a timer which checks if internal cache has changed before writing to file. Similar process is followed during PT persistence. This is however an implementation detail and can be decided upon during implementation itself.
```
I think this paragraph fits better in the "proposed solution" section of the proposal.

Reply:
Well, i am not proposing any change due to these points. But there might be some questions around if this has been considered as well. Hence i mentioned it there in the Additional Comments section.

I have corrected the typos.